### DISCUSSION TITLE
cloud-carbon-footprint / ccf-backstage-plugin

### DISCUSSION THREAD

**Discussion by Satya7340** [cloud-carbon-footprint/cloud-carbon-footprint#1107] [Q&A]
Hi Team,

Facing error while trying to install CCF plugin in backstage frontend.
I ran below command -

cd packages/app 
yarn add @cloud-carbon-footprint/backstage-plugin-frontend

<img width="905" alt="CCF" src="https://user-images.githubusercontent.com/39889634/229989835-2450b6b1-a125-429b-ad0a-d828aa3a7b48.png">

**Comment by 4upz** [comment]
Hi @Satya7340 thanks for opening this discussion! I see that you're getting assistance in the Google Group, so I'll close this thread in favor of finding a solution there: https://groups.google.com/g/cloud-carbon-footprint/c/SCpZ34E54VA
